### PR TITLE
chore: 替换 bundlesize 为 size-limit

### PR DESCRIPTION
**Pull request by LiHaoGit**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |  替换 bundlesize 为 size-limit         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

原因 #36574

### CODE REVIEW

**Comment by afc163** [comment]
别替换，先双跑。

**Comment by zombieJ** [comment]
替换的上下文是什么？

**Comment by afc163** [comment]
我猜是 https://github.com/ant-design/ant-design/discussions/36574

File: package.json [hunk 1]
```diff
@@ -282,6 +284,7 @@
     "scrollama": "^3.0.0",
     "semver": "^7.3.5",
     "simple-git": "^3.0.0",
+    "size-limit": "^7.0.8",
```
Comment (afc163) [inline package.json:287]: 没加 github action，PR 里看不到效果。

图片

**Pull request merged**